src/Zir.zig

```diff
@@ -2782,6 +2783,29 @@ pub const Inst = struct {
         index: u32,
     };
 
+    pub const SwitchBlockErrUnion = struct {
+        operand: Ref,
+        bits: Bits,
+        main_src_node_offset: i32,
+
+        pub const Bits = packed struct(u32) {
+            /// If true, one or more prongs have multiple items.
+            has_multi_cases: bool,
+            /// If true, there is an else prong. This is mutually exclusive with `has_under`.
```

Comment:
this comment looks like it was copy pasted and no longer applies since there is no `has_under` field.